Resources/Prototypes/Entities/Clothing/Uniforms/color_jumpskirts.yml

```diff
@@ -366,12 +366,12 @@
         color: "#ffe14d"
       - state: trinkets-equipped-INNERCLOTHING
 
-# Purple Jumpskirt
+# Lilac Jumpskirt - CD
 - type: entity
   parent: ClothingUniformSkirtBase
-  id: ClothingUniformJumpskirtColorPurple
-  name: purple jumpskirt
-  description: A generic light purple jumpskirt with no rank markings.
+  id: ClothingUniformJumpskirtColorLilac
```

Comment:
I don't like this "id injection" you are doing very much. I would prefer if this could remain the same as it is on wizden and have a different name for the new jumpsuit. If we are going to go with this it needs comments on both of the prototypes to make it clear what is going on.

Reply:
so, keep the ID on the lilac jumpskirt as ClothingUniformJumpskirtPurple, and make the other like ClothingUniformJumpskirtDarkPurple?

Reply:
Yes, that is preferable.

Reply:
adressed